src/azure-cli/azure/cli/command_modules/acr/_client_factory.py

```diff
@@ -31,7 +32,7 @@ def cf_acr_registries(cli_ctx, *_):
 
 
 def cf_acr_registries_tasks(cli_ctx, *_):
-    return get_acr_service_client(cli_ctx).registries
+    return get_acr_service_client(cli_ctx, VERSION_2019_06_01_PREVIEW).registries
```

Comment:
Use VERSION_2017_10_GA instead of VERSION_2019_06_01_PREVIEW if you want to use 2017-10-01 api version.

Reply:
I think there is a bit of a confusion here, so let me clarify.

ACR CLI supports two different but related RPs. One is ACR RP and another is ACR Tasks RP. Tasks RP is sort of under ACR, but we maintain two different swaggers, SDKs, API versions.

ACR is currently on VERSION_2017_10_GA
ACR Tasks is currently on VERSION_2019_06_01_PREVIEW (with this PR)

There were some tests previously that grabbed models from another unexpected version of ACR. That has been fixed by regenerating the test records (as you can see from the PR).

However, some tests still fail. Could you help me understand if those errors belong to us?